### DISCUSSION TITLE
The jenkins admin website of milvus.io can access without authorization through the public network

### DISCUSSION THREAD

**Discussion by SpikeWong** [milvus-io/milvus#14648] [Q&A and General discussion]
https://ci.milvus.io:18080/jenkins/ , I can simply operate these pipelines without authorization, not sure is there any security risk?

**Comment by yhmo** [comment]
Yes,  but changing the authorization way now has risks of breaking lots of things, so we plan to do it after  GA release.   We have already deployed Jenkins on the DevOps cluster, but have issues that need to work out.
